WordPress/Classes/ViewRelated/Plugins/PluginViewController.swift

```diff
@@ -88,4 +96,11 @@ class PluginViewController: UITableViewController {
         tableView.endUpdates()
     }
 
+    override func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        // This is a hack/work-around to remove the gap from the top of the tableView — the system leaves
+        // a gap with a `grouped` style by default — we want the banner up top, without any gaps.
+        guard section != 0 else { return 0 }
```

Comment:
Please, promote the `return 0` to the next line. [Ref. here](https://github.com/wordpress-mobile/swift-style-guide#braces).